WordPress/src/main/res/values/strings.xml

```diff
@@ -2280,7 +2280,9 @@
     <string name="support_ticket_subject">WordPress for Android Support</string>
 
     <string name="contact_support_input_placeholder">Send a message…</string>
+    <string name="contact_support_first_message_hi">Hi there, I\'m the Jetpack AI Assistant.</string>
     <string name="contact_support_first_message">What can I help you with?</string>
+    <string name="contact_support_first_message_hint">If I can\'t answer your question, I\'ll help you open a support ticket with our team!</string>
```

Comment:
Wouldn't it be better to use `\n\n` in these resources, not in the code?

```suggestion
    <string name="contact_support_first_message">Hi there, I\'m the Jetpack AI Assistant.\n\nWhat can I help you with?\n\nIf I can\'t answer your question, I\'ll help you open a support ticket with our team!</string>
```

Reply:
I did it that way at first.  However, If you may recall from previous discussions.  That would affect translations, the reason for separating them into their sentences.